DANS-KNAW/easy-convert-bag-to-deposit
# EASY-2780: no EASY-User-Account in bag-info.txt

Fixes EASY-2780: no EASY-User-Account in bag-info.txt

When applied it will
--------------------
* Remove EASY-User-Account from bag-info.txt
* recalculate on tagmanifest, not the payloads
* 

Where should the reviewer @DANS-KNAW/easy start?
------------------------------------------------

How should this be manually tested?
-----------------------------------

run unit tests and compare (select each pair in Intellij, CMD-D) `bag-info.text` and `tagmanifest-sha1.txt` of 
* src/test/resources/bags/01/04e638eb-3af1-44fb-985d-36af12fccb2d/bag-revision-1
* target/test/AppSpec/ingest-dir/04e638eb-3af1-44fb-985d-36af12fccb2d/bag-revision-1

See also [todo](https://github.com/DANS-KNAW/easy-convert-bag-to-deposit/blob/b024f4c9b81b8e6740f6a0bf10ab812b23c03e4f/src/test/scala/nl.knaw.dans.easy.bag2deposit/AppSpec.scala#L59-L58) in the code

Related pull requests on github
-------------------------------

repo                       | PR                | note
-------------------------- | ----------------- | ----
easy-                      | [PR#](PRlink)     | ..
